flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/KeyedStream.scala

```diff
@@ -54,28 +54,28 @@ class KeyedStream[T, K](javaStream: KeyedJavaStream[T, K]) extends DataStream[T]
   // ------------------------------------------------------------------------
 
   /**
-    * Applies the given [[TimelyFlatMapFunction]] on the input stream, thereby
+    * Applies the given [[ProcessFunction]] on the input stream, thereby
     * creating a transformed output stream.
     *
     * The function will be called for every element in the stream and can produce
     * zero or more output. The function can also query the time and set timers. When
     * reacting to the firing of set timers the function can emit yet more elements.
     *
```

Comment:
The function will be called for every element in the input streams and can produce zero or more
    * output elements. Contrary to the [[KeyedStream#flatMap]] function, this function can also
    * query the time and set timers. When reacting to the firing of set timers the function can directly
    * emit elements and/or register yet more timers.